pkg/splunk/client/enterprise.go

```diff
@@ -22,8 +22,13 @@ import (
 	"net/http"
 	"regexp"
 	"time"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+// kubernetes logger used by splunk.enterprise package
+var log = logf.Log.WithName("splunk.client")
```

Comment:
same above

Reply:
I am thinking we will leave it(we can have one per package in case anyone wants to use it?)